lib/temporal/client.rb

```diff
@@ -372,6 +372,25 @@ def list_closed_workflow_executions(namespace, from, to = Time.now, filter: {})
       fetch_executions(:closed, { namespace: namespace, from: from, to: to }.merge(filter))
     end
 
+    # Fetch a workflow's cron schedule
+    #
+    # @param namespace [String]
+    # @param workflow_id [String]
+    # @param run_id [String] If not specified, the most recent run will be used
+    #
+    # @return [String] the schedule in crontab format, or nil if there is no schedule
+    def get_cron_schedule(namespace, workflow_id, run_id: nil)
+      history_response = connection.get_workflow_execution_history(
```

Comment:
Can we filter it down to a single start event? Either via event type or max page size? Feels quite inefficient otherwise (history can be really long)